config/config-sample.yaml

```diff
@@ -5,10 +5,27 @@ bridge:
   domain: "localhost"
   # HomeServer URL
   homeserverUrl: "http://localhost:8008"
-  # MX inbound domain. 
-  # This is the domain to which inbound mails are sent by email users
-  mxDomain: "localhost"
-  # Mail submission port (587 is recommended)
-  mailPort: 2525
   # The TCP port on which the appservice runs on.
   port: 8090
+  # MX inbound domain.
+  # This is the domain to which inbound mails are sent by email users (eg: matrix.org)
+  mxDomain: "localhost"
+  # SMTP inbound port(Usually 25).
+  mailPort: 2525
+  # SMTP outbound port(Usually 25).
+  outboundPort: 2500
+  # SMTP next hop address (If you want to relay mails through some server. Leave empty, for no relay)
+  smtpHost: 'localhost'
+  # Set to True for enabling DKIM signature for outbound emails.
+  dkimEnabled: false
+  # DKIM private key file location (eg: './dkim-private.pem').
+  # Empty string for no DKIM signing (dkimEnabled = false)
+  dkimPrivateKeyLocation: ''
+  # DKIM selector name. Leave empty for no DKIM
+  dkimSelector: ''
+  # Enable Opportunistic TLS support for outbound emails.
+  startTLS: false
+  # TLS Key file location
```

Comment:
If any of these are optional, I'd encourage you to put a hash before them and prefixthe description with "Optional: ". Helps us admins out